### ISSUE TITLE
Follow system theme

### ISSUE THREAD

**Issue by Jnss98b** [williamckha/spicetify-fluent#67]
### 📝 Provide a description of the new feature

Basically use the light mode when system is in light mode and dark when the system is in dark mode

**Comment by rlaphoenix** [comment]
I don't know if this can be done with spicetify, but even if it can it may not be wanted by the user. They may specifically want an alternate theme to their system.

**Comment by rlaphoenix** [comment]
That method does seem to require patching of the Spotify exe which will complicate things, especially if users do not want to modify the exe.

However, more problematic is the Fluent light theme is very experimental and doesn't work very well even on supported Spotify versions. It will need to be worked on before this is implemented.

**Comment by Jnss98b** [comment]
I'm glad it's considered anyway

**Comment by nimsandu** [comment]
is this feature really useful?
i mean user can change config to light theme using shell
and that process is a one time process 
making things complicated will kill the calmness of fluent designing system 

special note- @williamckha is a great creator i respect him and even im using his theme as my themes base
i still following his principles

**Comment by Jnss98b** [comment]
It is useful if you have auto dark mode. It's gonna get tiring really fast to open up the terminal twice a day every day to change the theme

**Comment by nimsandu** [comment]
@Jnss98b Yah that makes sense 
But this should go smoothly as a step by step process 

First we have to develop light theme more fluent and less buggy

**Comment by Jnss98b** [comment]
Yeah I get that. There's no rush at all. I was just answering @nimsandu
